### PR TITLE
flowNavigation hover label renders jsx not strings->html

### PR DESCRIPTION
**Pull request by kaada**
flowNavigation hover label prefers jsx over a string being evaluated to html

### CODE REVIEW

File: src/flowNavigation/FlowNavigation.js [hunk 1]
```diff
@@ -91,8 +91,7 @@ FlowNavigation.propTypes = {
     Types.shape({
       label: Types.string.isRequired,
       onClick: Types.func,
-      hoverLabel: Types.string,
-      hoverHTML: Types.bool,
+      hoverLabel: Types.oneOfType([Types.element, Types.string]),
```
Comment (oliverviljamaa) [inline src/flowNavigation/FlowNavigation.js]: I believe we should just allow `Types.node` (`string` or `element`, basically), and create a breaking (major version change) release 😊 What do you think?

Comment (kaada) [reply src/flowNavigation/FlowNavigation.js]: Oh, yea. That's a good idea. Fixing.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# v12.0.0
+## flowNavigation hover label renders jsx not strings->HTML
+flowNavigation hover label no longer accepts rendering strings->HTML. hoverHTML-flag is removed.
```
Comment (oliverviljamaa) [inline CHANGELOG.md]: Please check that docs are up to date as well (for example, `FlowNavigation.docs` seems to still use `hoverHTML`) 😊

Comment (kaada) [reply CHANGELOG.md]: :panic:

**Pull request merged**